### ISSUE TITLE
Create a Basic Checklist Page

### ISSUE THREAD

**Issue by mwi-pharju** [magicwrighter/checklistdojo#4]
Before we can do all our fancy meta checklist management, we're going to need to start with a normal checklist.

We'll want it to look something like this:
![image](https://user-images.githubusercontent.com/45949550/50517575-ea9fa980-0a7e-11e9-9299-a97ad0834dc5.png)

Breaking this down, the checklist page should have the following:
- Title
- Description
- A list of checklist items
-- Whether it's checked
-- The name of the item
- A means of adding a new checklist item
- A means of deleting a checklist item
- A button that can be used to mark a checklist as "complete" (this won't do anything, yet)
